docs/api/ethereum.md

```diff
@@ -3203,13 +3249,18 @@ Multi-order actions define the atomic operations of the actions order.
 |-|-|-
 | CREATE_ASSET | 1 | Create a new asset.
 | UPDATE_ASSET_IMPRINT | 4 | Update asset imprint.
+| SET_ABILITIES | 5 | Sets abilities.
 | TRANSFER_ASSET | 2 | Transfer an asset.
 | TRANSFER_VALUE | 3 | Transfer a value.
 
 ::: Warning
 There is a possibility of unintentional behavior where asset imprint can be overwritten if more than one `UPDATE_ASSET_IMPRINT` order per asset is active. Be aware of this when implementing.
 :::
 
+::: Warning
+There is a possibility of unintentional behavior where account abilitites can be overwritten if more than one `SET_ABILITIES` order per account is active. Be aware of this when implementing.
```

Comment:
There is a possibility of unintentional behavior where account abilities could be overwritten if more than one `SET_ABILITIES` order per account is active. Be aware of this when implementing.